generate.ts

```diff
@@ -205,6 +205,7 @@ function generate() {
   const rootDirFiles = readdirSync(rootDir).filter((file) => !file.startsWith("."));
   if (rootDirFiles.length === 1 && rootDirFiles[0] === "rtag.yml") {
```

Comment:
Need to modify generate to create the proper directory structure for a java project so files don't have to be manually copied.